src/loader.rs

```diff
@@ -462,31 +477,77 @@ impl<'b, 'w> Loader<'b, 'w> {
-                let texture: Result<_, anyhow::Error> = self.load_texture(texture).await;
+                let ty = "fbx_color_texture";
+                let texture: Result<_, anyhow::Error> = self.load_texture(texture, ty).await;
                 texture.context("Failed to load diffuse texture")?
             }),
             None => None,
         };
+        let normal_map = match normal_map(&material_obj) {
+            Some(texture) => Some({
+                let ty = "fbx_normal_map";
+                let texture: Result<_, anyhow::Error> = self.load_texture(texture, ty).await;
+                texture.context("Failed to load normal map")?
+            }),
+            None => None,
+        };
+
+        let emissive_texture = match emissive_map(&material_obj) {
+            Some(texture) => Some({
+                let ty = "fbx_emissive_map";
+                let texture: Result<_, anyhow::Error> = self.load_texture(texture, ty).await;
+                texture.context("Failed to load emissive texture")?
+            }),
+            None => None,
+        };
+
+        // See discussion on how to interpret specular color:
+        // TODO convert to PBR
+        // https://google.github.io/filament/Material%20Properties.pdf
+        let _specular_colors = match specular_colors(&material_obj) {
+            Some(texture) => Some({
+                let ty = "fbx_specular_colors";
+                let texture: Result<_, anyhow::Error> = self.load_texture(texture, ty).await;
+                texture.context("Failed to load specular colors")?
+            }),
+            None => None,
+        };
+
+        // TODO: occlusion texture
 
         let properties = material_obj.properties();
+
         let shading_model = properties
             .shading_model_or_default()
             .context("Failed to get shading model")?;
-        let mut material = match shading_model {
-            ShadingModel::Lambert | ShadingModel::Phong => {
-                // TODO: convert shading model to PBR, see
-                // https://github.com/Sagoia/FBX2glTF/blob/dc300136c080c2f206b447ed15fb73e942653120/src/gltf/Raw2Gltf.cpp#L255
-                // and following code
-                StandardMaterial::default()
-            }
-            v @ ShadingModel::Unknown => bail!("Unknown shading model: {:?}", v),
+
+        // TODO: add the `RAW_SHADING_MODEL_PBR_MET_ROUGH` variant to ShadingModel enum
+        // Based on FBX2glTF code
+        // https://github.com/Sagoia/FBX2glTF/blob/dc300136c080c2f206b447ed15fb73e942653120/src/gltf/Raw2Gltf.cpp#L255
+        let (metallic, roughness) = if matches!(shading_model, Lambert | Phong) {
+            let shininess = properties.shininess()?;
+            let roughness = shininess.map(|s| (2.0 / (2.0 + s)).sqrt());
+            (0.4, roughness.unwrap_or(0.8))
+        } else {
+            warn!("Encountered an unknown shading_model in {label}, the resulting material may be unexpected");
```

Comment:
I opted to replace the `bail!` with a `warn!`, however we lose compilation failure if new variants are added in a future update of `fbxcel-dom`